newIDE/app/src/ObjectsRendering/PixiResourcesLoader.js

```diff
@@ -217,6 +218,37 @@ export default class PixiResourcesLoader {
     return fontFamily;
   }
 
+  /**
+   * Get the the data from a bitmap font (fnt/xml) resource in the IDE.
+   */
+  static getResourceBitmapFont(
+    project: gdProject,
+    resourceName: string
+  ): Promise<any> {
+    if (!project.getResourcesManager().hasResource(resourceName))
+      return Promise.reject(
+        new Error(`Can't find resource called ${resourceName}.`)
+      );
+
+    const resource = project.getResourcesManager().getResource(resourceName);
+    if (resource.getKind() !== 'bitmapFont')
+      return Promise.reject(
+        new Error(
+          `The resource called ${resourceName} is not a bitmap font file. Require .fnt or .xml format.`
+        )
+      );
+
+    const fullUrl = ResourcesLoader.getResourceFullUrl(project, resourceName, {
+      isResourceForPixi: true,
+    });
+
+    if (!fullUrl) {
+      return Promise.resolve('Arial');
```

Comment:
We might want to check what should be returned by default if the resource is found, maybe reject the promise. ("Arial" seems not what we seek here, it's a string, while we should return a `.fnt` or `.xml` file content).